packages/crayons-core/src/components/data-table/custom-cells/anchor/custom-cell-anchor.tsx

```diff
@@ -0,0 +1,20 @@
+import { Component, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'fw-custom-cell-anchor',
+  styleUrl: 'custom-cell-anchor.scss',
+  shadow: true,
+})
+export class CustomCellUser {
```

Comment:
Why is the class named `CustomCellUser`?

Reply:
Named this as `CustomCellUser` to let users know that this displays user's information in a custom cell. kindly let me know if you feel we can improve on this with a more appropriate name.

Reply:
Shouldnt this be "CustomCellAnchor" ?

Reply:
Oh! my bad. Didn't notice the mistake before. I have changed this. Thanks.